qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/diagnostics/QuteDiagnosticsInExpressionWithIfSectionTest.java

```diff
@@ -120,6 +120,14 @@ public void notOperator() {
 		testDiagnosticsFor(template, d);
 	}
 
+	@Test
+	public void onlyNotOperator() {
+		String template = "{#if !}{/if}";
+		Diagnostic d = d(0, 0, 0, 0, QuteErrorCode.SyntaxError, "Unsupported param type: NOT",

```

Comment:
It would be nice to put the error in the right place, if it's not too difficult to manage. In the case Fred submitted, there were spaces at the start of the line, so in that case the error is highlighting a space character.

Reply:
We could do that but it will work only for the first #if which have thoses errors. And if there is an error before the #if, the qute parser will not report it.

I created an issue for adjust error range at https://github.com/redhat-developer/quarkus-ls/issues/770 but as soon as Qute parser doesn't support fault tolerant, we should avoid spending some times to adjust error range. 

I think we should ignore this error and manage this error in our qute diagnostics, but let's wait for the state of the Qute parser before working on SyntaxError.